github/Organization.py

```diff
@@ -469,16 +469,27 @@ def get_issues(self, filter=github.GithubObject.NotSet, state=github.GithubObjec
             url_parameters
         )
 
-    def get_members(self):
+    def get_members(self, filter=github.GithubObject.NotSet,
```

Comment:
`filter` is a python built-in function, it sucks but it would be preferable if we could use another word >_<